src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -87,6 +87,14 @@
     <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(RuntimeIdentifier)' == 'linux-x64' ">
+    <PackageReference Include="ServerlessSecuritylinux-x64" Version="*-*" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(RuntimeIdentifier)' != 'linux-x64' ">
```

Comment:
```suggestion
  <ItemGroup Condition=" '$(RuntimeIdentifier)' != 'win-x64' ">
```

Reply:
It is not equal comparison, i.e. any other runtimes win-x64 or win-x86 pick the ServerlessSecuritywin-x64 NuGet.